### DISCUSSION TITLE
Feature request:

### DISCUSSION THREAD

**Discussion by evdemin** [algolia/instantsearch#5470] [Ideas]
Hi! I am trying to impement Algolia in Quasar Framework. I have a q-select box where I can type in e.g. some addresses. I want Algolia to look for the input in the data bank. If there are some hits I want them to be added to the list of q-select options.

**Comment by Haroenv** [comment]
You can use the slot of ais-menu for that, as in the docs here: https://www.algolia.com/doc/api-reference/widgets/menu/vue/#widget-param-default

**Reply by evdemin** [reply]
I have tried this way but items is an empty array despite the fact that Algolia retrieves correct data from the backend 

      <ais-instant-search :search-client="searchClient" index-name="venues" ref="instantSearch">
    <ais-search-box>
      <template #default="{ currentRefinement,refine }">
              <q-input
                standout="text-secondary"
                outlined
                dark
                square
                type="search"
                :model-value="currentRefinement"
                :label="t('search.search')"
                class="fit"
                @update:model-value="(query)=>debouncedSearch(query,refine )"
              >
              </q-input>
            </template>
    </ais-search-box>
    <ais-menu attribute="hits"  >
  
      <template  v-slot="{items}">
      <select class="fit">
    
          <option v-for="item in items" >
            {{ item}}
          </option>
        </select>
      </template>
    </ais-menu>
  </ais-instant-search>

**Reply by Haroenv** [reply]
Are you use the attribute is `hits`? wouldn't it be the name of a specific facet?

**Reply by Haroenv** [reply]
Maybe you're actually looking for `ais-hits`, not `ais-menu`?

**Reply by evdemin** [reply]
Yes, it's helped! Thanks a lot!
